contents/blog/mobile-session-replay.md

```diff
@@ -0,0 +1,144 @@
+---
+title: How we built mobile replay
+date: 2024-09-18
+author:
+ - ian-vanagas
+ - manoel-aranda-neto
+rootpage: /blog
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/posthog-engineering-blog.png
+featuredImageType: full
+tags:
+ - Guides
+---
+
+[Session replay](/session-replay) is one of the most powerful tools for understanding user behavior. Web session replay has been a core part of PostHog for a long time now (it was built in our first hackathon), but mobile teams have had to wait longer.
```

Comment:
Not sure if we have a link to our hackathon blogs somewhere. Would be nice to add if we do.
```suggestion
[Session replay](/session-replay) is one of the most powerful tools for understanding user behavior. Web session replay has been a core part of PostHog for a long time now (it was built in our [first hackathon]()), but mobile teams have had to wait longer.
```